src/appforms/tests/tests/core/040-Model07FieldSelect.js

```diff
@@ -0,0 +1,51 @@
+describe("Dropdown Field Model", function() {
+
+  function getMockDrowdownFieldData(include_blank_option){
+    return {
+      "required": true,
+      "type": "dropdown",
+      "name": "Dropdown",
+      "fieldCode": null,
+      "_id": "572b7c7c398524923d6343ac",
+      "adminOnly": false,
+      "fieldOptions": {
+        "validation": {
+          "validateImmediately": true
+        },
+        "definition": {
+          "options": [
+            {
+              "label": "Option 1",
+              "checked": false
+            },
+            {
+              "label": "Option 2",
+              "checked": false
+            }
+          ],
+          "include_blank_option": include_blank_option
+        }
+      },
+      "repeating": false
+    };
+  }
+
+  it("Get Options, no blank option included", function() {
+    var dropdownField = new appForm.models.Field(getMockDrowdownFieldData(true));
+
+    var dropdownOptions = dropdownField.getDropdownOptions();
+    assert.equal(3, dropdownOptions.length, "Expected 3 dropdown options");
+    //The first label should be an empty string if the blank option is included
+    assert.equal("", dropdownOptions[0].label);
+  });
+
+  it("Get Options, blank option included", function() {
+    var dropdownField = new appForm.models.Field(getMockDrowdownFieldData(false));
+
+    var dropdownOptions = dropdownField.getDropdownOptions();
+    assert.equal(2, dropdownOptions.length, "Expected 2 dropdown options");
+    //The first label should be an empty string if the blank option is included
```

Comment:
Perhaps this comment should be removed as it seems more related to the previous test?